package.json

```diff
@@ -2072,25 +2072,33 @@
         ]
     },
     "scripts": {
-        "lint": "eslint --ext .ts src",
-        "format": "prettier --ignore-path .gitignore --ignore-path .prettierignore --write .",
+        "lint:ts": "eslint --ext .ts,.cjs scripts src test_fixtures",
+        "lint:lua": "",
```

Comment:
lua linters like `luacheck` are available but are not provided through npm unfortunately. Requiring contributors to install an external tool would be annoying but not out of the question.